packages/graph/hash_graph/tests/rest-test.http

```diff
@@ -19,11 +19,10 @@ Accept: application/json
     client.test("status", function() {
         client.assert(response.status === 200, "Response status is not 200");
     });
-    client.global.set("text_data_type_id", response.body.version_id);
 %}
 
 ### Get Text data type
-GET http://127.0.0.1:3000/data-type/{{text_data_type_id}}
+GET http://127.0.0.1:3000/data-type/https%3A%2F%2Fblockprotocol.org%2F%40blockprotocol%2Ftypes%2Fdata-type%2Ftext%2Fv%2F1
```

Comment:
May I suggest we do this instead:
```suggestion
GET http://127.0.0.1:3000/data-type/{{encodeURIComponent("https://blockprotocol.org/@blockprotocol/types/data-type/text/v/1")}}
```
could you check if this works with the jetbrains IDE?
It'll make it easier to glance over :)

Reply:
Sadly, it does not work. However, it's possible to assign a variable in the statement above:
```
> {%
    client.test("status", function() {
        client.assert(response.status === 200, "Response status is not 200");
    });
    client.global.set("text_data_type_uri", encodeURIComponent(response.body.inner["$id"]));
%}

### Get Text data type
GET http://127.0.0.1:3000/data-type/{{text_data_type_uri}}

> {%
    client.test("status", function() {
        client.assert(response.status === 200, "Response status is not 200");
    });
%}
```

Reply:
Okay, I think we should do that for the test that previously depended on the globals to be set

Reply:
Applied in https://github.com/hashintel/hash/pull/838/commits/8fcb33ee8372ec111a8fb05ff99cec334672c2da